docs/NMOS With JPEG XS.md

```diff
@@ -23,20 +23,45 @@ and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119]
+### Sources
 
-If the Receiver supports streams meeting the traffic shaping and delivery timing requirements for ST 2110-22, it SHOULD use the `urn:x-nmos:cap:transport:st2110_21_sender_type` parameter constraint.
+The Source resource MUST indicate `urn:x-nmos:format:video` for the `format`.
+Source resources can be associated with many Flows at the same time.
+The Source is therefore unaffected by the use of JPEG XS compression.
 
-An example Receiver resource is provided in the [Examples](../examples/).
+### Flows
 
-## JPEG XS IS-04 Senders, Flows and Sources
+The Flow resource MUST indicate `video/jxsv` in the `media_type` attribute, and `urn:x-nmos:format:video` for the `format`.
+This has been permitted since IS-04 v1.1.
 
-Nodes capable of transmitting JPEG XS video streams MUST have Sender, Flow and Source resources in the IS-04 Node API.
+For Nodes implementing IS-04 v1.3 or higher, the following additional requirements on the Flow resource apply.
+
+In addition to those attributes defined in IS-04 for all coded video Flows, the following attributes defined in the [Flow Attributes register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/) of the [NMOS Parameter Registers][] are used for JPEG XS.
+
+- [Components](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#components)  
+  The Flow resource MUST indicate the color (sub-)sampling using the `components` attribute.
+  The `components` array value corresponds to the `sampling`, `width` and `height` values in the SDP format-specific parameters defined by RFC 9134.
+- [Profile](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#profile)  
+  The Flow resource MUST indicate the JPEG XS profile, which defines limits on the required algorithmic features and parameter ranges used in the codestream.
+  The permitted `profile` values are strings, defined as per RFC 9134.
+- [Level](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#level)  
+  The Flow resource MUST indicate the JPEG XS level, which defines a lower bound on the required throughput for a decoder in the image (or decoded) domain.
+  The permitted `level` values are strings, defined as per RFC 9134.
+- [Sublevel Bits Per Pixel](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#sublevel-bits-per-pixel)  
+  The JPEG XS sublevel defines a lower bound on the required throughput for a decoder in the codestream (or coded) domain.
+  When the `sublevel_bpp` attribute is omitted, the Full (or Unrestricted) sublevel is implied.
+  Otherwise, the `sublevel_bpp` value SHOULD indicate the compression ratio used by the encoder in bits per pixel as a number.
+  When the precise value is not available, the value MUST be set based on the sublevel indicated in the codestream or in the SDP format-specific parameters defined by RFC 9134.
```

Comment:
What does this mean?

Reply:
I think our intention is that if you encoded at 2.5 bpp, you _should_ put `"sublevel_bpp": 2.5`, but if that information isn't available at the NMOS layer you _must_ at least take the sublevel that willl be indicated in the codestream (and hopefully SDP data) such as `Sublev4bpp` and put in the corresponding values here, like `"sublevel_bpp": 4`.

Does that make sense? And if so, how should the explanation be improved?

Reply:
OK, if it's just representing the sublevel, I'm not entirely sure why we selected to use a number rather than enumerated strings like profile and level. I also thought we were accounting for implementations that were constrained more tightly, e.g. 1.5-4bpp.